src/js/components/question-input/question-input.js

```diff
@@ -0,0 +1,175 @@
+import Preview from "./preview";
+import "./question-input.scss";
+
+import { SimpleSelect } from "react-selectize";
+import { buildOptions } from "../../../lib/helpers/questions-helpers";
+
+export const TEXTAREA_PLACEHOLDER = "Enter your line here.";
```

Comment:
for these, you don't have to add export. can just be 
`const TEXTAREA_PLACEHOLDER = .....`

usually you export if you need to access that from another file